db/seeds.rb

```diff
@@ -1,7 +1,42 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+Book.create([
```

Comment:
Usually, when we are populating the db using seeds we want it to fail in case the record cannot be created successfully so that we are sure that the seed file creates things right. For that, you can use the exclamation mark (!) after the create (e.g., create!(...) ).

Reply:
Good, thanks for the tip.